internal/db/db_trace.go

```diff
@@ -0,0 +1,20 @@
+package db
+
+import (
+	"context"
+
+	"github.com/jackc/pgx/v5"
+	"github.com/sirupsen/logrus"
+)
+
+// Used for pgx logging with context information
+type dbTracer struct{}
+
+func (dt *dbTracer) TraceQueryStart(ctx context.Context, conn *pgx.Conn, data pgx.TraceQueryStartData) context.Context {
+	logrus.WithContext(ctx).Debug("Executing SQL", "sql", data.SQL, "args", data.Args)
+	return ctx
+}
+
+func (dt *dbTracer) TraceQueryEnd(ctx context.Context, conn *pgx.Conn, data pgx.TraceQueryEndData) {
+	// no-op
+}
```

Comment:
Custom tracer that logs SQL statements, very useful. The level is DEBUG so we can turn it off for production if needed.